MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.h

```diff
@@ -75,6 +78,45 @@ extern EFI_DRIVER_SUPPORTED_EFI_VERSION_PROTOCOL  gNvmExpressDriverSupportedEfiV
 
 #define NVME_MAX_QUEUES  3                              // Number of queues supported by the driver
 
+//
+// FormatNVM Admin Command LBA Format (LBAF) Mask
+//
+#define NVME_LBA_FORMATNVM_LBAF_MASK  0xF
+
+//
+// NVMe Completion Queue Entry Bits, Fields, Masks
+//
+#define NVME_CQE_STATUS_FIELD_MASK                       0xFFFF0000
+#define NVME_CQE_STATUS_FIELD_OFFSET                     16
+#define NVME_CQE_STATUS_FIELD_SCT_MASK                   0x0E00
+#define NVME_CQE_STATUS_FIELD_SCT_OFFSET                 0x9
+#define NVME_CQE_STATUS_FIELD_SC_MASK                    0x1FE
+#define NVME_CQE_STATUS_FIELD_SC_OFFSET                  0x01
+#define NVME_CQE_SCT_GENERIC_CMD_STATUS                  0x0
+#define NVME_CQE_SCT_CMD_SPECIFIC_STATUS                 0x1
+#define NVME_CQE_SCT_MEDIA_DATA_INTEGRITY_ERRORS_STATUS  0x2
+#define NVME_CQE_SCT_PATH_RELATED_STATUS                 0x3
+#define NVME_CQE_SC_SUCCESSFUL_COMPLETION                0x00
+#define NVME_CQE_SC_INVALID_CMD_OPCODE                   0x01
+#define NVME_CQE_SC_INVALID_FIELD_IN_CMD                 0x02
+
+#define NVME_ALL_NAMESPACES  0xFFFFFFFF
+
+// MU_CHANGE [BEGIN] - Support alternative hardware queue sizes in NVME driver

```

Comment:
This change came from a different commit https://github.com/microsoft/mu_basecore/commit/9837db39da115a1fa4a322d13adafbba862212d8.

Reply:
Should be fixed in latest, thanks for catching it.